apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2023-01-10T21:58:28Z"
-  build_hash: 1b20baf45a0b73a11b296050322a384c705fa897
-  go_version: go1.17.5
-  version: v0.22.0
-api_directory_checksum: ee32acc4d4a0ba7e2823dd20fdbe2c4ef1d9e0f4
+  build_date: "2023-02-06T20:33:47Z"
+  build_hash: c6651c200ba136e5c7f50ad8be751fae060a38e6
+  go_version: go1.19
+  version: v0.22.0-1-gc6651c2
```

Comment:
Please do a `git fetch --tags --all` in your `code-generator` directory to get the latest version information of the code generator

Reply:
Okay~